test/integration/resource-links-test.js

```diff
@@ -0,0 +1,137 @@
+import test from 'blue-tape'
+import * as contentful from '../../lib/contentful'
+
+/* Test data structure:
+ *
+ * Space 1: en-US, de, fr
+ * Space 2: en-US, de
+ * Space 3: en-US
+ *
+ * -- Space 1: Article
+ *   -- title
+ *   -- content -- local link --> Space 1: Content
+ *     -- text
+ *     -- additionalInfo -- xspace link --> Space 3: Additional Info
+ *       -- categories
+ *       -- date
+ *   -- author -- xspace link --> Space 2: Author
+ *     -- name (localized)
+ *     -- picture -- xspace link --> Space 3: Avatar
+ *       -- image
+ *       -- caption
+ *     -- book -- local link --> Space 2: Book
+ *       -- title
+ *   -- views -- xspace link --> Space 3: Views
+ *     -- count
+ */
+
+const client = contentful.createClient({
+  accessToken: '2O_YlI7Spwm16XXw-VZw9vxIIlUOeHLuzWwKXn3KMvo',
+  space: 'z788fz497ijs',
+  additionalTokens: {
+    mbfd18am5xhd: 'tZO0vih_LPefnfNhCDARaRjfiSNXLeJMYRvFe7U-6Yk',
+    p9sqh626r1fo: 'CagovFaZwA7Dta0chjKBU2iU4ASND0TLgCgQyep-Xnk'
+  }
```

Comment:
are those real tokens or mocked ones?

Reply:
Real CDA tokens to the testing space. This is how integration tests are currently working here.